vcl_templates/www.vcl.erb

```diff
@@ -250,7 +250,41 @@ sub vcl_recv {
       set req.http.GOVUK-ABTest-EducationNavigation = "A";
     }
   }
-  
+
+  <% 5.times do |i| %>
```

Comment:
This is an arbitrary number chosen to represent five 'slots' which teams can use for ab testing. This can be larger or smaller depending on what the teams feel is required.